config/optional/clamav.settings.yml

```diff
@@ -1,13 +1,13 @@
 enabled: true
 outage_action: 0
 overridden_schemes: {  }
-scan_mode: 1
+scan_mode: 0
```

Comment:
Is this the setting that puts it into TCP/IP mode?

Reply:
Yes